GOVERNANCE.md

```diff
@@ -0,0 +1,49 @@
+# Project Governance (Very Minimal Governance model)
+
+This open source project is managed by a Steering Committee composed of the maintainers of this project. Maintainers are
+defined as individuals with [maintain-level permissions on the data.all repository](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization).
+
+## Steering Committee
+
+The Steering Committee will be responsible for oversight of all technical, project, approval, and policy matters for the
+project. This notably includes brand and trademark management.
+
+The Steering Committee members are listed in the MAINTAINERS.md file in the repository. New maintainers (and accordingly,
+Steering Committee members) may be added or removed as outlined in [RESPONSIBILITIES.md](./RESPONSIBILITIES.md). The
+Steering Committee will appoint a Chair responsible for organizing Steering Committee activity. If the Steering Committee
+Chair is removed from the Committee (or the Chair steps down from that role), it is the responsibility of the Steering
+Committee to appoint a new Chair.
+
+The Steering Committee may, at its discretion, add or remove members who are not maintainers.
+
+## Voting
+
+The Steering Committee will strive for all decisions to be made by consensus. While explicit agreement of the entire
+Steering Committee is preferred, it is not required for consensus. Rather, the Steering Committee will determine
+consensus based on their good faith consideration of a number of factors, including the dominant view of the Steering
+Committee and nature of support and objections. The Steering Committee will document evidence of consensus in accordance
+with these requirements. If consensus cannot be reached, the Steering Committee will make the decision by a vote.
+
+The Steering Committee Chair will call a vote with reasonable notice to the Steering Committee, setting out a discussion
+period and a separate voting period. Any discussion may be conducted in person or electronically by text, voice, or video.
```

Comment:
Is there a time frame for discussion/voting?

Reply:
When the chair calls for a vote, then the discussion period is set.  This is helpful as it puts a deadline on making a decision and avoids "analysis paralysis"  The length of time depends on the complexity of the decision being made.

Reply:
Should we have a place where we record decisions? just GitHub issues? can we have like a record of decision issues? or tag them as "decision issues"